packages/diracx-web-components/src/components/shared/SearchBar/defaultFunctions.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+
+import type { SearchBarTokenEquation, InternalFilter } from "../../../types";
+
+/**
+ * Function to convert token equations to internal filters
+ * @param equations The token equations to convert
+ * @param setFilters The function to set the filters state
+ */
+export function convertAndApplyFilters(
+  equations: SearchBarTokenEquation[],
+  setFilters: React.Dispatch<React.SetStateAction<InternalFilter[]>>,
+) {
+  /**
+   * Mapping of operator labels to internal filter operators.
+   */
+  const operators = {
+    "=": "eq",
+    "!=": "neq",
+    ">": "gt",
+    "<": "lt",
+    "is in": "in",
+    "is not in": "not in",
+    like: "like",
+    "in the last": "last",
+  };
```

Comment:
I see you have `operators`  here and you have the opposite in `Utils.tsx`.
So may be you can create a `src/types/operators.tsx`, that would contain 1 or more enums around that, and then you would have functions to go from `=` to `eq` and vice versa.